qrcodereaderview/src/main/java/com/google/zxing/client/android/camera/AutoFocusManager.java

```diff
@@ -44,8 +43,9 @@
 
   private static final String TAG = AutoFocusManager.class.getSimpleName();
 
-  private static final long AUTO_FOCUS_INTERVAL_MS = 5000L;
+  protected static final long DEFAULT_AUTO_FOCUS_INTERVAL_MS = 5000L;
```

Comment:
I need this protected, to access this default value from other classes in the same package. 
For example: https://github.com/dlazaro66/QRCodeReaderView/pull/64/files#diff-63e2aacd2af702b6dfe82ecf928463efR52